standard-integration/server.js

```diff
@@ -1,34 +1,149 @@
-import "dotenv/config"; // loads variables from .env file
-import express from "express";
-import * as paypal from "./paypal-api.js";
-const {PORT = 8888} = process.env;
+import express from 'express';
+import fetch from 'node-fetch';
+import 'dotenv/config';
+import path from 'path';
 
+const { PAYPAL_CLIENT_ID, PAYPAL_CLIENT_SECRET } = process.env;
+const base = 'https://api-m.sandbox.paypal.com';
 const app = express();
 
-app.use(express.static("public"));
-
 // parse post params sent in body in json format
 app.use(express.json());
 
-app.post("/my-server/create-paypal-order", async (req, res) => {
```

Comment:
💯 this tripped me up the other day thinking `my-server` was not an actual route...